tests/types/raku-test.json

```diff
@@ -0,0 +1,277 @@
+{
+  "$schema": "https://packageurl.org/schemas/purl-test.schema-1.0.json",
+  "tests": [
+
+    {
+      "description": "valid Raku dist name without api",
+      "test_group": "base",
+      "test_type": "build",
+      "input": {
+        "type": "raku",
+        "namespace": "zef:lizmat",
+        "name": "SBOM::CycloneDX",
```

Comment:
@sjn @lizmat Just to be double sure before merging this... Raku dists are the problem target for a PURL?